tensorflow/python/training/basic_session_run_hooks.py

```diff
@@ -36,9 +36,12 @@
 import six
 
 from tensorflow.core.framework.summary_pb2 import Summary
+from tensorflow.core.protobuf import config_pb2
```

Comment:
Since all the other changes are reverted in this file, do we need these modifications?